src/Plugin/HistoryPlugin.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Http\Client\Common\Plugin;
 
+use Exception;
```

Comment:
Conflicts with the other exception import. Also, our convention is not to import root namespaces classes, but use them directly.